--- php-predis-0.6.6/test/PredisShared.php~	2011-10-20 12:45:42.000000000 +0300
+++ php-predis-0.6.6/test/PredisShared.php	2011-10-20 12:46:47.993123318 +0300
@@ -1,7 +1,7 @@
 <?php
 // -------------------------------------------------------------------------- //
 
-define('I_AM_AWARE_OF_THE_DESTRUCTIVE_POWER_OF_THIS_TEST_SUITE', false);
+define('I_AM_AWARE_OF_THE_DESTRUCTIVE_POWER_OF_THIS_TEST_SUITE', true);
 
 // -------------------------------------------------------------------------- //
 
@@ -17,7 +17,7 @@
 
 class RC {
     const SERVER_HOST      = '127.0.0.1';
-    const SERVER_PORT      = 6379;
+    const SERVER_PORT      = 6380; // non-standard port, so shouldn't destroy
     const DEFAULT_DATABASE = 15;
 
     const WIPE_OUT         = 1;
