--- php-predis-0.6.6/test/PredisShared.php~	2011-04-01 21:36:41.000000000 +0300
+++ php-predis-0.6.6/test/PredisShared.php	2011-10-20 21:06:27.088554381 +0300
@@ -1,7 +1,7 @@
 <?php
 // -------------------------------------------------------------------------- //
 
-define('I_AM_AWARE_OF_THE_DESTRUCTIVE_POWER_OF_THIS_TEST_SUITE', false);
+define('I_AM_AWARE_OF_THE_DESTRUCTIVE_POWER_OF_THIS_TEST_SUITE', true);
 
 // -------------------------------------------------------------------------- //
 
@@ -26,9 +26,11 @@
     return create_function($param, $function);
 }
 
+define('REDIS_SERVER_PORT', getenv('REDIS_SERVER_PORT'));
+
 class RC {
     const SERVER_HOST      = '127.0.0.1';
-    const SERVER_PORT      = 6379;
+    const SERVER_PORT      = REDIS_SERVER_PORT;
     const DEFAULT_DATABASE = 15;
 
     const WIPE_OUT         = 1;
